aiven/client/cliarg.py

```diff
@@ -93,6 +93,10 @@ def wrapped(self):
 arg.retention_bytes = arg("--retention-bytes", type=int, help="Retention limit in bytes (default: unlimited)")
 arg.service_name = arg("name", help="Service name")
 arg.service_type = arg("-t", "--service-type", help="Type of service (see 'service list-types')")
+arg.ns_name = arg("ns_name", help="Namespace name")
+arg.ns_type = arg("--ns-type", help="Namespace type ('aggregated' or 'unaggregated')", required=True)
+arg.ns_ret = arg("--ns-ret", help="Namespace retention period", required=True)
+arg.ns_res = arg("--ns-res", help="Namespace resolution")
```

Comment:
nit: perhaps `retention` and `resolution` instead of the abbreviated argument names would be a bit more readable?

Reply:
ok, changed